Externals/mbedtls/CMakeLists.txt

```diff
@@ -172,8 +172,6 @@ endfunction(link_to_source)
 
 string(REGEX MATCH "Clang" CMAKE_COMPILER_IS_CLANG "${CMAKE_C_COMPILER_ID}")
 
-include(CheckCCompilerFlag)
```

Comment:
This is used in the branch right below this, isn't it?

Reply:
The only use of `check_c_compiler_flag` that I can find is in CMake/CheckAndAddFlag.cmake which already has the include?

Reply:
Whoops, case sensitivity bit me